Updated the CRUD of reference data from v1.3

mickaelandrieu opened the pull request:
Hello, this is part of the cookbooks/tutorial to help a beginner create a reference data and be able to manage it in the back office. (see https://github.com/akeneo/pim-docs/issues/488)

Note this pull request depends on https://github.com/akeneo/pim-docs/pull/487 as we need to install CustomEntity Bundle and understand what it provides.

> Needed to be cherry picked in ``1.7``, ``1.6`` and ``1.5`` branches.

/!\ don't merge it => should be done in ``1.5`` branch

mickaelandrieu commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+====================================================
+
+.. note::
+    The code inside this cookbook entry requires you to install the :doc:`/reference/bundles/custom_entity_bundle/index`.
```
this is the link to the previous pull request, when merged :)

mickaelandrieu commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+                    name:
+                        data_name: rd.name
+                default:
+                    code: '%oro_datagrid.extension.orm_sorter.class%::DIRECTION_ASC'
```
What is missing here is a reference documentation about grid link, see https://github.com/akeneo/pim-docs/issues/490

MarieMinasyan commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+--------
+
+To benefit from the grid component (which comes natively with filtering and sorting),
+you can define the vendor grid as following:
```
_as follows:_

MarieMinasyan commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+
+.. note::
+
+    Want to know more about forms? Take a look at the `Symfony documentation`_.
```
_Want to learn more_

MarieMinasyan commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+------------------------
+
+Now we have created the grid and the required form for both creation and update,
+we only need to declare the reference data as "custom entities":
```
as a "custom entity".

MarieMinasyan commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+
+    You can define the same form type for both creation and edition tasks.
+
+From this point a working grid screen should be visible at ``/app_dev.php/enrich/color``.
```
_At this point, a working grid should be visible at_

MarieMinasyan commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+
+From this point a working grid screen should be visible at ``/app_dev.php/enrich/color``.
+
+Create an entry point in Back Office
```
_in the Back office_

MarieMinasyan commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+------------------------------------
+
+Most of the time, we want to make the screens available for our customers. Fortunately, this is
+really easy to add a new menu entry in back office:
```
_it is really easy to_

MarieMinasyan commented on cookbook/ui_customization/create_a_reference_data_crud.rst:
```diff
@@ -0,0 +1,176 @@
+
+.. note::
+
+    Want to know more about the menu management? Take a look at the :doc:`/cookbook/catalog_structure/creating_a_reference_data` cookbook.
```
_learn_ instead of _know_

mickaelandrieu replied:
dully noted :)

mickaelandrieu commented:
@MarieMinasyan sorry, closed in favor of https://github.com/akeneo/pim-docs/pull/494

Thanks for review, I've addressed your comments and completed my pull request
